force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1028,10 +1046,12 @@ export default class GeFormRenderer extends LightningElement{
         let validationErrorLabelReplacements = [dataImportHelper.donationDonorValue, dataImportHelper.donationDonorLabel];
 
         if (dataImportHelper.donationDonorValue === DONATION_DONOR.isAccount1) {
-            if (dataImportHelper.isAccount1ImportedPresent)
+            if (dataImportHelper.isAccount1ImportedPresent && !this._isInvalidDonorSelected)
                 validationErrorLabelReplacements.push(GeFormService.getFieldLabelByDevNameFromTemplate(DONATION_DONOR_FIELDS.account1ImportedField));
-            if (dataImportHelper.isAccount1NamePresent)
+            if (dataImportHelper.isAccount1NamePresent && !this._isInvalidDonorSelected)
                 validationErrorLabelReplacements.push(GeFormService.getFieldLabelBySourceFromTemplate(DONATION_DONOR_FIELDS.account1NameField));
+            if (this._isInvalidDonorSelected)
```

Comment:
If you move this if block to the top of the group of 3 if blocks, you can remove the redundant tests for !this._isInvalidDonorSelected